src/services/JobService.js

```diff
@@ -81,13 +81,29 @@ async function _validateSkills (skills) {
   }
 }
 
+/**
+ * Validate by exact match on name field if all skills exist.
+ *
+ * @param {Array} skills the list of skill names to validate
+ * @returns {Array} the list of matches skill objects
+ */
+async function _validateAndGetSkillsByNames(skillNames) {
+  const skills = await helper.getSkillsByExactNames(skillNames)
+  if (skills.length !== skillNames.length) {
```

Comment:
Let's let this code breathe a little -- whitespace is free.

```
async function _validateAndGetSkillsByNames(skillNames) {
  const skills = await helper.getSkillsByExactNames(skillNames)

  if (skills.length !== skillNames.length) {
    const foundSkillNames = skills.map(skill => skill.name)
    const notFoundSkillNames = _.difference(skillNames, foundSkillNames)
    throw new errors.BadRequestError(`Invalid skills: [${notFoundSkillNames}]`)
  }

  return skills
}
```

Reply:
Fixed